City_CRUD_API/crud.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from City_CRUD_API import models
+from City_CRUD_API.schemas import CityCreate
+
+
+async def get_all_cities(db: AsyncSession):
+    result = await db.execute(select(models.City))
+    return result.scalars().all()
+
+
+async def create_city(db: AsyncSession, city: CityCreate):
+    db_city = models.City(
+        name=city.name,
+        additional_info=city.additional_info,
+    )
+    db.add(db_city)
+    await db.commit()
+    await db.refresh(db_city)
+
+    return db_city
+
+
+async def get_city_by_id(db: AsyncSession, city_id: int):
+    result = await db.execute(
+        select(models.City).filter(models.City.id == city_id)
+    )
+    return result.scalars().first()
```

Comment:
In `get_city_by_id`, you might want to handle the case where the city is not found more explicitly, perhaps by raising an exception or returning a specific error message.